docs/releases/4.0.txt

```diff
@@ -725,6 +736,9 @@ Miscellaneous
 * The ``django.contrib.gis.admin.GeoModelAdmin`` and ``OSMGeoAdmin`` classes
   are deprecated. Use :class:`~django.contrib.admin.ModelAdmin` and
   :class:`~django.contrib.gis.admin.GISModelAdmin` instead.
+* Since form rendering now uses the template engine, the undocumented
+  ``BaseForm._html_output()`` helper method and the ability to return a ``str``
+  from ``ErrorList`` is deprecated.
```

Comment:
I have gone with. 

``` 
* Since form rendering now uses the template engine, the undocumented
  ``BaseForm._html_output()`` helper method is deprecated.

* The ability to return a ``str`` from ``ErrorList`` and ``ErrorDict`` is
  deprecated. It is expected these methods return a ``SafeString``.